com.cyberbotics.Webots.yaml

```diff
@@ -0,0 +1,53 @@
+app-id: com.cyberbotics.Webots
+runtime: org.freedesktop.Sdk
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: webots
+separate-locales: false
+base: io.qt.qtwebengine.BaseApp
```

Comment:
this is wrong. This BaseApp is only used in conjuction with the KDE runtime. And I believe 6.5 uses 22.08 as a base.